build-settings/babel.config.js

```diff
@@ -1,11 +1,22 @@
 /* eslint-disable import/no-commonjs */
 module.exports = {
-    presets: ["@babel/preset-flow", "@babel/preset-react", "@babel/preset-env"],
+    presets: [
+        "@babel/preset-flow",
+        "@babel/preset-react",
+        [
+            "@babel/preset-env",
+            {
+                useBuiltIns: "usage",
```

Comment:
What are the file size implications of this change? I'm worried that we're going to be shipping many more poly folks for every WB component that we have.

Reply:
Good call.  I was copying the webapp config without thinking.  After reading the docs, I think the default value of `false` is what we want.

Reply:
Doing so didn't seem to have an impact on the size of the dist file produced for wonder-blocks-button.

Reply:
Ok - so it's possible that it's already getting bundled then!